assets/js/experience.js

```diff
@@ -1,3 +1,12 @@
+// text animation effect
+
+var _window = window,Splitting = _window.Splitting,ScrollOut = _window.ScrollOut;
```

Comment:
Instead of adding this everywhere add this in the style.css which is common for all pages so that code redundancy can be avoided

Reply:
> Instead of adding this everywhere add this in the style.css which is common for all pages so that code redundancy can be avoided

@anushbhatia I have removed the code redundancy now.